test/utilities.jl

```diff
@@ -65,6 +65,13 @@ end
     @test OuterModule in Documenter.Utilities.submodules(OuterModule)
     @test OuterModule.InnerModule in Documenter.Utilities.submodules(OuterModule)
     @test length(Documenter.Utilities.submodules(OuterModule)) == 2
+
+    @test Documenter.Utilities.isabsurl("file.md") === false
+    @test Documenter.Utilities.isabsurl("../file.md") === false
+    @test Documenter.Utilities.isabsurl(".") === false
+    @test Documenter.Utilities.isabsurl("https://example.org/file.md") === true
+    @test Documenter.Utilities.isabsurl("http://example.org") === true
+    @test Documenter.Utilities.isabsurl("ftp://user:pw@example.org") === true
```

Comment:
That's actually a relative URL technically, but a test would be useful nevertheless.

The interpretation of such an URL should be that they're relative to the root of the document (and not relative to the filesystem root). I added a special case for that to the LaTeX code as well.